src/com/generic/main/Shipment.java

```diff
@@ -0,0 +1,48 @@
+package com.generic.main;
+
+public class Shipment {
+	private String shipmentID;
+	private String freight;
+	private float weight;
+	private int receiptDate; //Need to figure out the date format
+	
```

Comment:
I think we need to add shipment_method as a field here.
From what I understand, shipment_method is not a type of shipment, it is a label for a shipment
(E.G. a shipment "has-a" shipment method).

We probably dont want to write 100 subclass of Shipment for 100 shipment_method when all the subclass contains the same state and member fields. (Just as the letter A is a Character but we dont create a subclass of Character called A)

But if you choose to go this route, then Shipment must be an interface so that the List<Shipment> from the Warehouse class can reference all the sub classes of Shipment